src/common/components/modal-dialog/modal-dialog.component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from './modal-dialog.styles.module.css';
+
+export interface Props {
+  children: React.ReactNode;
+  isOpen: boolean;
+}
+
+export const ModalDialog: React.FC<Props> = props => {
+  const { children, isOpen } = props;
+  const [open, setOpen] = React.useState(isOpen);
+  const handleClick = () => {
+    setOpen(false);
+  };
+
+  return (
+    open && (
+      <div className={styles.container} role="presentation">
+        <div className={styles.dialog} role="dialog">
+          <button className={styles.dialogButton} onClick={handleClick}>
```

Comment:
Nice start, it's time to start making design decisitions :)

Replace the close button by a given icon, choose an icon library for that O:-)

Plus add some styling

Reply:
Change icon, but I don't add more styles, better to wait for high definition designs.